lib/spoom/cli.rb

```diff
@@ -27,6 +28,11 @@ class Main < Thor
       desc "bump", "bump Sorbet sigils from `false` to `true` when no errors"
       subcommand "bump", Spoom::Cli::Commands::Bump
 
+      desc "--version", "show version"
+      def __print_version
```

Comment:
The `__` prefix make it unaccessible as a command like `spoom __print_version` and avoid having creating the `no_command do .. end` block that Sorbet doesn't like.